_includes/shared/markup/ccloud/config-create.adoc

```diff
@@ -0,0 +1,9 @@
+Create a development configuration file at `configuration/dev.properties` with your Confluent Cloud information.
```

Comment:
1. Should this be ccloud.properties?
2. Does `configuration` folder exist yet at this point?

Reply:
1. yes and fixed. 2. It doesn't... Instead of having that `ccloud.properties` file floating around, I moved the `mkdir configuration` earlier in the steps. I think the flow is pretty okay. See commit https://github.com/confluentinc/kafka-tutorials/pull/786/commits/71cceceb46a091134d731263f01929e75b9e687c